src/components/controlbuttonsbar/ControlButtonsBar.tsx

```diff
@@ -8,14 +8,17 @@ const ControlButtonsBar = (): JSX.Element => {
 	const controlButtonsBar =
 		useAppSelector((state) => state.settings.controlButtonsBar);
 	const hideSelfView = useAppSelector((state) => state.settings.hideSelfView);
+	const browser = useAppSelector((state) => state.me.browser);
```

Comment:
You could simplify here and say:
`const isMobile = useAppSelector((state) => state.me.browser.platform === 'mobile');`

Reply:
Done